migrate_pim/upgrade/upgrade_from_40_to_50.rst

```diff
@@ -29,6 +29,52 @@ You have to make sure your system components are updated to the version required
 
     So there's no need to export and reimport data for this system.
 
+
+Updated System dependencies
+---------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+TODO?
+
+Enterprise Edition
+^^^^^^^^^^^^^^^^^^
+The following libraries must be added to your operating system:
+
+.. code:: bash
+
+    $ apt-get install aspell-it aspell-sv aspell-da aspell-nl aspell-no aspell-pt-br
+
+Updated crontab definition
+--------------------------
+
+Community Edition
+^^^^^^^^^^^^^^^^^
+
+Added
```

Comment:
Indeed crontab change is new in the upgrade documentation... I wonder if it makes sense...
Furthermore you missed the "cron wrapper", 
pim:data-quality-insights:prepare-evaluations are duplicate commands in community and enterprise

Reply:
I don't know what you mean by "cron wrapper" - nothing related to Data quality insights feature.

regarding the `pim:data-quality-insights:prepare-evaluations` yes it as to be added to the crontab if you're migrating both CE or EE version, so it as to be documented that way.

Reply:
and for me it make sense, if the fact that their are new cron to configure from a version to another is not in a migration guide so you've to tell the customer/integrator: play the 7 differences game between you're installed PIM and the "Install a pim from scratch" documentation to find what is new, what is deleted, what is updated ;)

We can discuss about that IRL ;)